src/Interpreters/Context.cpp

```diff
@@ -2250,6 +2332,28 @@ void Context::initializeTraceCollector()
     shared->initializeTraceCollector(getTraceLog());
 }
 
+#if USE_ROCKSDB
+void Context::initializeMergeTreeMetadataCache(const String & dir, size_t size)
+{
+    rocksdb::Options options;
+    rocksdb::BlockBasedTableOptions table_options;
+    rocksdb::DB * db;
+
+    options.create_if_missing = true;
+    options.statistics = rocksdb::CreateDBStatistics();
+    auto cache = rocksdb::NewLRUCache(size);
```

Comment:
Do we really need this LRU cache? As I understood from this [link](https://github.com/EighteenZi/rocksdb_wiki/blob/master/Memory-usage-in-RocksDB.md) it's the in-memory cache for uncompressed blocks in RocksDB. But we need metadata cache only on startup and LRU cache is always dropped after restart. So, we will never have a hit to LRU cache.